lib/pf/Switch/Aruba/cx.pm

```diff
@@ -0,0 +1,219 @@
+package pf::Switch::Aruba::cx;
+
+=head1 NAME
+
+pf::Switch::Aruba::cx
+
+=head1 SYNOPSIS
+
+Module to manage rebanded Aruba HP CX Switche
+
+=head1 STATUS
+
+=over
+
+=item Supports
```

Comment:
Can you confirm that this switch indeed supports all the features that are outlined there (like dACL, RADIUS CLI login, etc)

Reply:
It's based on the Aruba::5400 who support that.